### PR TITLE
OVN: Use images built from source

### PR DESCRIPTION
**Pull request by mdbooth**
These images are also compatible with ovn-central-operator.

### CODE REVIEW

File: ansible/files/ocp/ovn/northd.yaml [hunk 1]
```diff
@@ -17,22 +17,22 @@ spec:
       containers:
       # ovn-northd: convert network objects in nbdb to flows in sbdb
       - name: northd
-        image: "quay.io/mbooth/ovn-central:2.11"
+        image: "quay.io/mbooth/ovn-central:trunk"
```
Comment (dprince) [inline ansible/files/ocp/ovn/northd.yaml:20]: As this gets polished you might just go ahead and push it to quay.io/openstack-k8s-operators?

**Review by dprince — APPROVED** [review]

**Comment by dprince** [comment]
LGTM

**Comment by luis5tb** [comment]
LGTM too

File: ansible/files/ocp/ovn/northd.yaml (hunk 1 above)
Comment (mdbooth) [reply ansible/files/ocp/ovn/northd.yaml:20]: Yes, absolutely.

**Pull request merged**